src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function (n1) {
+    return function (n2) {
+      count++;
+
+      if (count >= 4 && count % 2 === 0) {
```

Comment:
```suggestion
      if (count > 3 && count % 2 === 0) {
```
Так буде трішки більш інтуїтивно